src/js/jsx/Scrim.jsx

```diff
@@ -198,6 +230,11 @@ define(function (require, exports, module) {
             if (!nextState.activeDocumentInitialized || !nextState.recentFilesInitialized) {
                 return false;
             }
+            
+            if (this.state.isDragging === nextState.isDragging &&
```

Comment:
Overall, for shouldCOmponentUpdate functions, it's better to calculate when you *should* render, instead of when you shouldn't.